sklearn/tree/tree.py

```diff
@@ -108,6 +108,19 @@ def __init__(self,
         self.class_weight = class_weight
         self.presort = presort
 
+    def get_depth(self):
+        """Returns the depth of the decision tree, i.e. the maximum
```

Comment:
Pep 257:

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

No line breaks in first line.